ssv/validator_registration.go

```diff
@@ -60,6 +61,20 @@ func (r *ValidatorRegistrationRunner) ProcessPreConsensus(signedMsg *types.Signe
 		return nil
 	}
 
+	// only 1 root, verified in basePreConsensusMsgProcessing
+	root := roots[0]
+	// randao is relevant only for block proposals, no need to check type
```

Comment:
you are correct about Randao, but what is the point of putting this comment here?
I am missing some context?